docs/user-guide/admin-cli.md

```diff
@@ -9,14 +10,341 @@ by the Docker user via the Docker CLI. However, the Docker CLI is limited in wha
+    vm                  Add, removes and lists VMs in a tenant
+
+optional arguments:
+  -h, --help            show this help message and exit
+```
+### Create
+
+Creates a new named tenant and optionally assigns VMs.
+
+Sample:
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant create --name DevAppX
+tenant create succeeded
+```
+
+The tenant to VM association can be done at create time.
+
+Sample:
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant ls
+Uuid                                  Name     Description  Default_datastore  VM_list
+------------------------------------  -------  -----------  -----------------  ---------------------------
+dc17b3b1-1f4d-4f8b-a6f4-3dba064ba88f  tenant1               default_ds         photon.vsan.1,photon.vsan.2
+
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant create --name DevAppX --vm-list ubuntu
+tenant create succeeded
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant ls
+Uuid                                  Name     Description  Default_datastore  VM_list
+------------------------------------  -------  -----------  -----------------  ---------------------------
+dc17b3b1-1f4d-4f8b-a6f4-3dba064ba88f  tenant1               default_ds         photon.vsan.1,photon.vsan.2
+5afa4767-ecd8-489e-9d97-5c46ae50d3fc  DevAppX               default_ds         ubuntu
+
+[root@localhost:~]
+
+```
+
+#### Help
+```
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant create -h
+usage: vmdkops_admin.py tenant create [-h] --name NAME
+                                      [--vm-list vm1, vm2, ...]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --name NAME           The name of the tenant
+  --vm-list vm1, vm2, ...
+                        A list of VM names to place in this Tenant
+```
+
+### Access
+
+Change the access control for a tenant.
+This includes ability to control which datastores are accessible by a tenant and controlling what a
+tenant can do on a given datastore.
+
+#### Help
+```bash
+[root@localhost:~]  /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant access -h
+usage: vmdkops_admin.py tenant access [-h] {rm,add,set,ls} ...
+
+positional arguments:
+  {rm,add,set,ls}
+    rm             Remove all access to a datastore for a tenant
+    add            Add a datastore access for a tenant
+    set            Modify datastore access for a tenant
+    ls             List all access info for a tenant
+
+optional arguments:
+  -h, --help       show this help message and exit
+```
+#### Add
+
+Grants datastore access to a tenant.
+
+Sample:
+
+```bash
+/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py tenant access add --name tenant1 --datastore datastore1 --rights create,delete,mount
+{'mount_volume': 1, 'global_visibility': 0, 'max_volume_size': 0, 'create_volume': 1, 'delete_volume': 1, 'datastore': 'datastore1', 'usage_quota': 0}
```

Comment:
Is it possible for you to include the small fix in this PR too? Just remove  remove this print at Line 1008 (in function "tenant_access_add")? This should not affect any existing functionality. Just remove a unused print command which I added for debugging.